package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "market-site",
+  "version": "0.1.0",
+  "description": "market-site\r ===========",
+  "main": "index.js",
+  "scripts": {
+    "build:css": "tailwindcss build css/styles.css -o public/styles.css",
+    "build:css:dist": "env NODE_ENV=production tailwindcss build css/styles.css -o public/styles.css",
+    "start": "live-server public --no-browser --port 8080"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/eve-val/market-site"
+  },
+  "author": "Sam Tesla <samuel.tesla@gmail.com>",
+  "license": "ISC",
+  "bugs": {
```

Comment:
Same problem here with the URLs

Reply:
Hrm. What should we put there?

Reply:
I suppose we can probably omit author, tbh. But do we want something for license?